### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sprax2013**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+zoe is still work in progress. You can try running [the sandbox](https://github.com/NudelErde/zoe/releases) but please don't expect anything other than a simple demo.
+
+# Compiling
+I don't know why you would like to compile zoe at its current state, but...
+
+You can compile zoe [as we tell CircleCI to](https://github.com/NudelErde/zoe/blob/master/.circleci/config.yml).
+* Make sure you are using a Linux machine to compile and run zoe
+* Install all dependencies `apt install build-essential cmake libglew-dev libx11-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev libgl-dev`
+
+1. Clone the repository with all its submodules (Use `git submodule update --init --recursive` after cloning)
```
Comment (NudelErde) [inline README.md]: you could clone with `git clone --recurse-submodules https://github.com/NudelErde/zoe`

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
-# zoe
\ No newline at end of file
+# zoe [![CircleCI](https://img.shields.io/circleci/build/github/NudelErde/zoe?logo=circleci)](https://circleci.com/gh/NudelErde/zoe)
```
Comment (NudelErde) [inline README.md]: fossa badge?

File: README.md (hunk 1 above)
Comment (Sprax2013) [reply README.md]: Added

File: README.md (hunk 2 above)
Comment (Sprax2013) [reply README.md]: Added

**Pull request merged**